politeiawww/database/database.go

```diff
@@ -113,6 +113,12 @@ type User struct {
 	Deactivated                     bool      // Whether the account is deactivated or not
 	EmailNotifications              uint64    // Notify the user via emails
 
+	// Access times for proposals that have been accessed by the user. Each
+	// string represents a proposal token, and the int64 represents the time that
+	// the proposal has been most recently accessed in the format of a UNIX
+	// timestamp.
+	ProposalAccessTimes map[string]int64
```

Comment:
This name and comment are a bit misleading.  The access times are not being updated when a user accesses a proposal, they are being updated when a user accesses a proposal's comments.  At the very least you should update the comment to reflect this.  You may want to consider changing the name to something like `CommentsAccessTime`.